crates/ethereum/evm/src/lib.rs

```diff
@@ -32,20 +32,33 @@ pub mod dao_fork;
 pub mod eip6110;
 
 /// Ethereum-related EVM configuration.
-#[derive(Debug, Clone, Copy, Default)]
+#[derive(Debug, Clone, Default)]
```

Comment:
I wanna remove this here, this should flag where we forget to use the new function

```suggestion
#[derive(Debug, Clone)]
```